cmake/README.md

```diff
@@ -21,6 +21,38 @@ cmake ... -DENABLE_VAAPI=ON -DENABLE_VDPAU=OFF ...
 
 Unfortunately, Kodi's CMake gazillion options are not fully documented yet. For more information and an updated list of options, please check the main **[CMakeLists.txt](../CMakeLists.txt)**.
 
+## Buildsystem variables
+The buildsystem uses the following variables (which can be passed into it when executing cmake with the -D`<variable-name>=<value>` format) to manipulate the build process (see READMEs in sub-directories for additional variables):
+- `CMAKE_BUILD_TYPE` specifies the type of the build. This can be either *Debug* or *Release* (default is *Release*)
+- `CMAKE_INSTALL_PREFIX` points to the directory where the built add-ons and their additional files (addon.xml, resources, ...) will be installed to (defaults to `<ADDON_DEPENDS_PATH>`)
+- `CMAKE_TOOLCHAIN_FILE` can be used to pass a toolchain file into the add-on builds
+- `CORE_SOURCE_DIR` points to the root directory of the project (default is the absolute representation of ../../.. starting from this directory)
```

Comment:
`../../..` is left over from early cmake days, when CMakeLists.txt was located at project/cmake/CMakeLists.txt. Now the path would be `..`, and there's probably a better way of wording this.

Reply:
Oops responded to the wrong PR